db/migrate/20220517131300_change_tables_to_support_recrypt.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeTablesToSupportRecrypt < ActiveRecord::Migration[7.0]
+  def change
+    add_column :teams, :recrypt_state, :integer, default: 1, null: false
+    add_column :teams, :encryption_algorithm, :string, default: 'AES256', null: false
```

Comment:
is it wise to use this deprecated value as default?
maybe first add new column ... update all existing entries .. then make column not null